src/components/z-breadcrumb/index.tsx

```diff
@@ -0,0 +1,414 @@
+      if (this.pathsList[i].name.length > this.truncateChar) {
+        if (this.truncatePosition !== null) {
+          if (this.truncatePosition > 0) {
+            const arrayToPush = this.pathListCopy.splice(0, this.truncatePosition);
+            arrayToPush.forEach((item) => {
+              this.collapsedElements.push(item);
+            });
+            this.pathsList.splice(0, this.truncatePosition);
+            this.truncatePosition = 0;
+
+            return;
+          }
+          if (this.truncatePosition === 0) {
+            const arrayToPush = this.pathListCopy.splice(0, this.truncatePosition + 1);
+            arrayToPush.forEach((item) => {
+              this.collapsedElements.push(item);
+            });
+            this.pathsList.splice(0, this.truncatePosition + 1);
+            this.truncatePosition = null;
+
+            return;
+          }
+        }
+        if (i !== this.pathsList.length - 1) {
+          const truncatedString = this.truncateWithEllipsis(this.pathsList[i].name, this.truncateChar, null);
+          this.currentEllipsisText = this.pathsList[i].name;
+          this.pathsList[i].name = truncatedString;
+          this.pathsList[i].hasTooltip = true;
+          this.truncatePosition = i;
+
+          return;
+        }
+      }
+    }
+  }
+
+  private truncateWithEllipsis(str, length, ending): string {
+    if (length == null) {
+      length = 100;
+    }
+    if (ending == null) {
+      ending = "...";
+    }
+    if (str.length > length) {
+      return str.substring(0, length - ending.length) + ending;
+    }
+
+    return str;
+  }
+
+  private getPathsItemsList(): BreadcrumbPath[] | undefined {
+    if (!this.paths) {
+      return Array.from(this.hostElement.children).map((item: HTMLAnchorElement) => {
+        return {
+          name: item.textContent,
+          path: item.href,
+        };
+      });
+    }
+
+    return typeof this.paths === "string" ? JSON.parse(this.paths) : this.paths;
+  }
+
+  private renderMobileBreadcrumb(): HTMLDivElement {
+    const filteredPath = this.pathsList.filter((item) => !!item.path);
+    const lastPath = filteredPath[filteredPath.length - 1];
+
+    return (
+      <nav
+        aria-label="Breadcrumb"
+        class={{
+          underlined: this.pathStyle === BreadcrumbPathStyle.UNDERLINED,
+          semibold: this.pathStyle === BreadcrumbPathStyle.SEMIBOLD,
+        }}
+      >
+        <ol>{this.renderNode(lastPath, true)}</ol>
+      </nav>
+    );
+  }
+
+  private renderHomepageNode(item): HTMLLIElement {
+    return (
+      <li>
+        <a
+          class={{
+            "homepage-icon": this.homepageVariant === BreadcrumbHomepageVariant.ICON,
+            "homepage-text": this.homepageVariant === BreadcrumbHomepageVariant.TEXT,
+          }}
+          href={item.path}
+          onClick={(e) => this.handlePreventFollowUrl(e, item)}
+        >
+          {this.homepageVariant === BreadcrumbHomepageVariant.ICON ? (
```

Comment:
In entrambi i casi (icona o scritta) si potrebbe mattere direttamente il link a protocol+host, senza dover aggiungere il primo link a config.
In tal caso occhio ai conteggi del maxNodesToShow

Reply:
Io lascerei così, perchè avere il primo nodo relativo alla home è importante in ottica SEO, per permettere al crawler di seguire il link, quando si usano gli slot per popolare la breadcrumb.